main.cpp

```diff
@@ -19,15 +19,29 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
-
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter) {};
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        const auto *castExp = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `castExp` is a null pointer? How to protect the program from crashing in such cases?

Reply:
may be to add check for this